src/router/router.ts

```diff
@@ -1,109 +1,58 @@
 import { compile, match } from 'path-to-regexp'
 import type { Route, RouterSchema } from '~/types'
 import { getLocaleFactory } from '~/utils/locale-utils'
+import { StaticRouter } from './static-route' // Importing separate static router
 
 export class Router {
   private schema: RouterSchema
-  private static PAGE_HREF = '/'
 
   constructor(schema: RouterSchema) {
     this.schema = schema
   }
 
-  /**
-   * Gets relevant page href
-   */
-  public static getPageHref() {
-    return Router.PAGE_HREF
-  }
-
-  /**
```

Comment:
Can we keep these docs please? At least a comment part. It makes the code visually separated in my eyes.

Reply:
Hey, I thought those docs were on the code 🥲. Probably erased it accidentally. Will fix it along with the rest of the suggestions.